CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `WithoutTimestamps` option to `go.opentelemetry.io/otel/exporters/stdout/stdoutmetric` to sets all timestamps to zero. (#3828)
 - The new `Exemplar` type is added to `go.opentelemetry.io/otel/sdk/metric/metricdata`.
   Both the `DataPoint` and `HistogramDataPoint` types from that package have a new field of `Exemplars` containing the sampled exemplars for their timeseries. (#3849)
+- The New `Warn` function is added to `internal/global` package for internal logging. (#3900)
```

Comment:
This is probably not needed, the `internal` package is not accessible to the users we intend as the audience of this document. We should mention that the logging levels of the project have changed.

Reply:
Would it be more appropriate if I changed it to something like: 


- The logging levels of the project are updated as below: (#3900) 
  | Name | Severity |
  | -- | -- |
  | Error | 0 |
  | Warn | 1 |
  | Info | 4 |
  | Debug | 8 |

Reply:
```suggestion
- The internal logging introduces a warning level equal to `V(8)` (verbosity level) . (#3900)
```